tests/unit/moduleapi/propagate.tcl

```diff
@@ -198,7 +198,6 @@ tags "modules" {
                         {del asdf*}
                         {incr notifications}
                         {del asdf*}
-                        {config set maxmemory 1}
```

Comment:
please update the comment on the top of this block

Reply:
at first I removed it, but after a reflection I think maybe it's better to keep it that sometimes we and users can recall the problem by this comment.

Reply:
@soloestoy i already removed it from this PR: b25bd98342d90b517780cdfa447f7b8833b2a487
i'm not sure how this comment helps, but if we wanna keep it, we need to modify it.
the old one was outdated and misleading.
maybe something like:
```
# Note that although CONFIG SET maxmemory is called in this flow (see issue #10014),
# eviction will happen and will not induce propagation of the CONFIG command (see #10019).
```